.env.template

```diff
@@ -1,3 +1,2 @@
-DATABASE_URL=DATABASE_URL
```

Comment:
we dont really need to set this locally unless we using a non-standard db configuration....i.e changed default port 5432 etc.